app/assets/stylesheets/mobile/login-signup-page.scss

```diff
@@ -1,5 +1,23 @@
 // Shared styles
 
+html.keyboard-visible.mobile-view {
+  body.login-page,
+  body.signup-page,
+  body.invite-page {
+    height: calc(100dvh - var(--header-offset) - env(keyboard-inset-height));
+    max-height: calc(00dvh - var(--header-offset) - env(keyboard-inset-height));
+    overflow-y: scroll;
+    overflow-x: hidden;
+    padding-top: 0;
+
+    .login-fullpage,
+    .signup-fullpage,
+    .invites-show {
+      padding-top: var(--header-offset);
+    }
+  }
+}
+
```

Comment:
Do we need to do this because we're setting a `height: 100%` to the `login-fullpage/signup-fullpage` elements? I'm not sure that's all that useful on mobile.